Upstream: yes, https://github.com/networkx/networkx/commit/3987370c957de15249809672933dc06c2fed6fc1

Modified from upstream to not touch the not-in-pypi-existing file requirements/extras.txt
Required to fix the TestPylab test

From 3987370c957de15249809672933dc06c2fed6fc1 Mon Sep 17 00:00:00 2001
From: Jarrod Millman <jarrod.millman@gmail.com>
Date: Wed, 30 Oct 2019 05:22:12 -0700
Subject: [PATCH] Fix matplotlib deprecation (#3697)

Fixes #3694
---
 networkx/drawing/tests/test_pylab.py | 2 +-
 requirements/extras.txt              | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/networkx/drawing/tests/test_pylab.py b/networkx/drawing/tests/test_pylab.py
index 2ecd268d37..0fd3773b06 100644
--- a/networkx/drawing/tests/test_pylab.py
+++ b/networkx/drawing/tests/test_pylab.py
@@ -172,7 +172,7 @@ def test_multigraph_edgelist_tuples(self):
         # See Issue #3295
         G = nx.path_graph(3, create_using=nx.MultiDiGraph)
         nx.draw_networkx(G, edgelist=[(0, 1, 0)])
-        nx.draw_networkx(G, edgelist=[(0, 1, 0)], node_size=[10, 20])
+        nx.draw_networkx(G, edgelist=[(0, 1, 0)], node_size=[10, 20, 0])
 
     def test_alpha_iter(self):
         pos = nx.random_layout(self.G)
